.github/CONTRIBUTING.md

```diff
@@ -12,14 +12,16 @@ style, we make use of the [pre-commit](https://pre-commit.com/) framework to run
 several *hooks* when committing code. Please follow the instructions below if
 you intend to contribute to the lmfit repository:
 
-- clone the GitHub repository:
-  ``git clone https://github.com/lmfit/lmfit-py.git``
+- fork the GitHub repository
+- clone your forked GitHub repository:
+  ``git clone https://github.com/<your-name>/lmfit-py.git``
 - install all (optional) dependencies either using ``pip`` or ``conda``:
-  ``pip -r install requirements-dev.txt`` or
+  ``pip install -U -r requirements-dev.txt`` or
   ``conda install <packages in requirements-dev.txt>``
 - initialize ``pre-commit`` running ``pre-commit install`` in the lmfit directory
 - create a new branch: ``git checkout -b <awesome_new_feature>``
 - start coding
+- install the latest version of your code using `python3 setup.py install --user`
```

Comment:
the exact `python` command depends of course on which version you want to use. For example, on macOS `python3` refers to `/usr/bin/python3` and I typically don't want to use the system version of Python for doing stuff. 

So I guess I would prefer to use `python` here and if we want one could add a comment that this should be replaced by whatever version you want to use (same holds for `pip`, there you also don't say `pip3`). I think we can leave this to the end-user to figure out, right?

Reply:
I agree with @reneeotten.  Maybe that could be 
        "install the latest version of your code using the usual `python setup.py` commands"

... maybe someone wants to use `python setup.py develop`, for example.

Reply:
Thanks for noticing the `python3`. :) 

It was also a little bit strange to me that it was necessary to install it first which is not how the development workflow works, so I have changed it to `python -m pytest test`.

Reply:
I am not sure if this is completely correct. I think it will work without installing first for the tests, but am not so sure about the documentation (for docstrings and such it will likely use the installed version of `lmfit`.) So I would probably prefer to keep the original, but then again it might not be all that crucial.